### DISCUSSION TITLE
Question 2

### DISCUSSION THREAD

**Discussion by Zakiyyyy** [Zakiyyyy/my-repo#4] [Q&A] [answered]
How to add  javascript in my project?

**Comment by Kholik-Muzaki** [comment, marked as answer]
To add JavaScript scripts to a Bootstrap boilerplate, you can embed them within your HTML file. Make sure to include the <script> tags within the <body> element to ensure the scripts are executed after the page is loaded. You can place your custom JavaScript scripts after the Bootstrap JavaScript scripts to ensure proper dependencies. For instance:
After loading Bootstrap CSS in the <head> section and the main HTML content, you can include Bootstrap JavaScript and your custom JavaScript scripts at the bottom of the HTML file, just before the </body> tag. Make sure to adjust the paths to your custom scripts according to their location. This way, your scripts will load after the HTML page is fully loaded and immediately after Bootstrap JavaScript is executed.
